pkg/cli/image/mirror/mirror.go

```diff
@@ -268,173 +268,176 @@ func (o *MirrorImageOptions) Validate() error {
 func (o *MirrorImageOptions) Run() error {
 	var continuedOnFailure bool
 	start := time.Now()
-	p, err := o.plan()
+	plans, err := o.plan()
+	//p, err := o.plan()
```

Comment:
nit: probably drop this commented-out line

Reply:
thanks, updated!